packages/@glimmer/syntax/lib/v2-a/normalize.ts

```diff
@@ -644,14 +650,6 @@ class ElementNormalizer {
     let pathLoc = variableLoc.withEnd(pathEnd);
 
     if (isComponent) {
-      // If the component name is uppercase, the variable is not in scope,
-      // and the template is not in strict mode, run the optional
-      // `customizeComponentName` function provided as an option to the
-      // precompiler.
-      if (!this.ctx.strict && uppercase && !inScope) {
```

Comment:
This uppercase check is removed, but I don’t see it brought back. 

Can you add a test that confirms a template without a block param never resolves for `<rental></rental>`.

Reply:
Test added!